cmd/minikube/cmd/start.go

```diff
@@ -312,3 +350,73 @@ func init() {
 	viper.BindPFlags(startCmd.Flags())
 	RootCmd.AddCommand(startCmd)
 }
+
+// saveConfig saves profile cluster configuration in
+// $MINIKUBE_HOME/profiles/<profilename>/config.json
+func saveConfig(clusterConfig cluster.Config) error {
+	data, err := json.MarshalIndent(clusterConfig, "", "    ")
+	if err != nil {
+		return err
+	}
+
+	profilePath := filepath.Join(constants.GetMinipath(), "profiles", viper.GetString(cfg.MachineProfile))
```

Comment:
Can we add this as a "constant" in https://github.com/kubernetes/minikube/blob/master/pkg/minikube/constants/constants.go#L109? To make it a little easier to find.